client/src/components/pages/TeacherProfile/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import './index.css';
+import TeacherImage from '../../../assets/teacher.jpg';
+import fakeData from './fakeData';
+
+class TeacherProfile extends Component {
+  state = {
+    data: {},
+    active: 1,
```

Comment:
`activeContent` would be a little more descriptive name (other suggestions welcome)